app/views/users/_show.html.slim

```diff
@@ -0,0 +1,35 @@
+article.user-information class="user-id-#{user.id}"
```

Comment:
Oh I see now what you were doing above, rendering show within show. I'd tend to call this `users/_user.html.slim`...clearer what you mean, without the template name duplication.